### PR TITLE
coptic: Clean up play store listing

### PR DESCRIPTION
**Pull request by rillian**
Add screenshots and streamline the description.

Only an en-US localization available so far, but otherwise as far as I can tell coptic should now have a similar listing to other published addons.

### CODE REVIEW

File: addons/languages/coptic/apk/src/main/play/listings/en-US/full-description.txt [hunk 1]
```diff
@@ -1,10 +1,9 @@
-Coptic Language pack
+Coptic Language input for AnySoftKeyboard
```
Comment (menny) [inline addons/languages/coptic/apk/src/main/play/listings/en-US/full-description.txt:1]: Is there a benefit for using `input` rather than `pack`?

Comment (rillian) [reply addons/languages/coptic/apk/src/main/play/listings/en-US/full-description.txt:1]: No. It seemed more descriptive, but I'm happy to use *pack* if you prefer.

There's a lot of variation among the various language addons. The most common seem to be `$langauge layouts and dictionary [for AnySoftKeyboard]` and `$language Language pack` Do you have a preference between those?

Comment (menny) [reply addons/languages/coptic/apk/src/main/play/listings/en-US/full-description.txt:1]: Maybe `$language Language pack` is better, as it a more generic term.

Comment (menny) [reply addons/languages/coptic/apk/src/main/play/listings/en-US/full-description.txt:1]: could you change that? And also rebase the branch

Comment (rillian) [reply addons/languages/coptic/apk/src/main/play/listings/en-US/full-description.txt]: Ok, description changed and rebased against master.

**Comment by menny** [comment]
thank you!

**Comment by rillian** [comment]
Thanks for merging!

**Pull request merged**